password-rotation/README.md

```diff
@@ -27,14 +27,6 @@ module "password-rotation" {
   sheet_name_dev       = ""
   sheet_name_val       = ""
   sheet_name_prod      = ""
-
```

Comment:
My intention for the README was to show all the variable values, with those variables that have defaults commented out (I made a note of this above but it doesn't show in the diff), and those that don't with blank values so the user knows they need to fill them out.  Then the user can just copy-paste the example, and the commented-out variables will fall back to the default since they aren't overridden.  It sounds like this wasn't very clear, though, so I'm open to suggestions!
I would probably argue for consistency, that is, either remove all the defaults from the example, or keep them all in (in which case we would want to fix the default example for idm prod as you did in `variables.tf`)

Reply:
I like the idea of removing all the defaults from the README, since they will rarely change and they are not environment-specific. I think `event_rule_enabled` should be configurable at the top-level, since it is environment-specific.  I'll change `event_rule_enabled` default to `false` and set it to `true` at the top-level.